server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -122,17 +127,22 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
         // both to be safe. Empty sring denotes a cache that was cleared due to a service summary or the document
         // was created within a different tenant.
         if (document.deli === undefined || document.deli === null) {
-            context.log?.info(`New document. Setting empty deli checkpoint`, { messageMetaData });
+            const message = "New document. Setting empty deli checkpoint";
+            context.log?.info(message, { messageMetaData });
+            Lumberjack.error(message, { messageMetaData });
```

Comment:
Should this be `Lumberjack.info()` instead?

Also, let's use `getLumberBaseProperties()` instead of `{ messageMetaData }`.